src/git.md

```diff
@@ -0,0 +1,181 @@
+# Using git
+
+The Rust project uses [git][git] to manage its source code. In order to
+contribute, you'll need some familiarity with its features so that your changes
+can be incorporated into the compiler.
+
+[git]: https://git-scm.com
+
+The goal of this page is to cover some of the more common questions and
+problems new contributors face. Although some git basics will be covered here,
+if you have never used git or GitHub before you may find that this is still a
+little too fast for you. In that case, it would make sense to first read some
+introductions to git, such as the Beginner and Getting started sections of
+[this tutorial from Atlassian][atlassian-git]. GitHub also provides
+documentation and guides for beginners.
+
+[atlassian-git]: https://www.atlassian.com/git/tutorials/what-is-version-control
```

Comment:
```suggestion
[documentation] and [guides] for beginners.

[atlassian-git]: https://www.atlassian.com/git/tutorials/what-is-version-control
[documentation]: https://docs.github.com/en/github/getting-started-with-github/set-up-git
[guides]: https://guides.github.com/introduction/git-handbook/
```
You might also recommend https://git-scm.com/book/en/v2/; it's a little more in-depth but has great info.